src/coreclr/src/vm/threadsuspend.cpp

```diff
@@ -2726,25 +2698,23 @@ void Thread::PreWorkForThreadAbort()
     ResetUserInterrupted();
 
     if (IsRudeAbort()) {
-        if (HasLockInCurrentDomain()) {
-            AppDomain *pDomain = GetAppDomain();
-            // Cannot enable the following assertion.
-            // We may take the lock, but the lock will be released during exception backout.
-            //_ASSERTE(!pDomain->IsDefaultDomain());
-            EPolicyAction action = GetEEPolicy()->GetDefaultAction(OPR_ThreadRudeAbortInCriticalRegion, this);
-            switch (action)
-            {
-            case eExitProcess:
-            case eFastExitProcess:
-            case eRudeExitProcess:
-                    {
-                GetEEPolicy()->NotifyHostOnDefaultAction(OPR_ThreadRudeAbortInCriticalRegion,action);
-                GetEEPolicy()->HandleExitProcessFromEscalation(action,HOST_E_EXITPROCESS_ADUNLOAD);
-                    }
-                break;
-            default:
-                break;
-            }
+        AppDomain *pDomain = GetAppDomain();
+        // Cannot enable the following assertion.
+        // We may take the lock, but the lock will be released during exception backout.
+        //_ASSERTE(!pDomain->IsDefaultDomain());
```

Comment:
Everything's the default domain now right?

Reply:
Yes, that is more code that can be deleted #33073